OpenRA.Mods.Common/FileFormats/InstallShieldCABCompression.cs

```diff
@@ -303,6 +307,9 @@ public void CopyTo(Stream output, Action<int> onProgress)
 
 			public byte[] GetBytes(uint count)
 			{
+				if (currentVolume == null)
+					throw new FileNotFoundException($"Volume {currentVolumeID} is not available");
+
```

Comment:
Another bugfix, elsewhere if the `SetVolume` call threw on this line:
```c#
if (!volumes.TryGetValue(currentVolumeID, out currentVolume))
	throw new FileNotFoundException($"Volume {currentVolumeID} is not available");
```
then that method exits and leaves currentVolume null. So we're not safe in being able to assume currentVolume is non-null. These extra throws ensure any caller that tries again gets a proper error and not a NRE.

This is very hard to spot since it deals with an exception path, but the extra compiler warnings make it possible.